Client.xcodeproj/project.pbxproj

```diff
@@ -8443,6 +8492,17 @@
 					"$(BUILD_DIR)/Release$(EFFECTIVE_PLATFORM_NAME)",
 				);
 				GCC_PREPROCESSOR_DEFINITIONS = "DEBUG=1";
+				HEADER_SEARCH_PATHS = (
+					"$(SRCROOT)/ThirdParty/sqlcipher",
+					"$(inherited)",
+					/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include,
```

Comment:
This seems like a strange requirement, why do we need this?

Either way, if you really need it, it should be `$(DEVELOPER_DIR)/Toolchains/XcodeDefault.xctoolchain/usr/include`, no hard-coding paths for Xcode